pkg/accounting/data.go

```diff
@@ -8,8 +8,8 @@ var activeSeriesByCheckType = map[string]int{
 	"http_basic":     34,
 	"http_ssl":       122,
 	"http_ssl_basic": 38,
-	"ping":           81,
-	"ping_basic":     25,
+	"ping":           82,
+	"ping_basic":     26,
```

Comment:
we need to update these numbers in plugin, https://github.com/grafana/synthetic-monitoring-app/blob/9113ffb972b6a22bf5218d2db5e18a7708a6f4e4/src/checkUsageCalc.ts#L16